Code/Tools/TestImpactFramework/Runtime/Native/Code/Source/Target/Native/TestImpactNativeTargetListCompiler.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Artifact/TestImpactArtifactException.h>
+#include <Target/Native/TestImpactNativeTargetListCompiler.h>
+
+namespace TestImpact
+{
+    //! Compiles the production target artifacts and test target artifactss from the supplied build target artifacts and test target meta
```

Comment:
This has already been removed, it's been moved from the common lib to the native lib.